Documentation/ceph-disaster-recovery.md

```diff
@@ -336,3 +336,24 @@ Assuming `dataHostPathData` is `/var/lib/rook`, and the `CephCluster` trying to
 1. Bring the Rook Ceph operator back online by running `kubectl -n rook-ceph edit deploy/rook-ceph-operator` and set `replicas` to `1`.
 1. Watch the operator logs with `kubectl -n rook-ceph logs -f rook-ceph-operator-xxxxxxx`, and wait until the orchestration has settled.
 1. **STATE**: Now the new cluster should be up and running with authentication enabled. `ceph -s` output should not change much comparing to previous steps.
+
+## Adopting into a new Kubernetes cluster with PVCs
+
+It is possible to migrate/restore an rook/ceph cluster from an exising Kubernetes cluster to a new one without resorting to SSH access or ceph tooling. This allows doing the migration using standard kubernetes resources only. This guide assumes the following
+1. You have a CephCluster that uses PVCs to persist mon and osd data. Let's call it the "old cluster"
+1. You can restore the PVCs as-is in the new cluster. Usually this is done by taking regular snapshots of the PVC volumes and using a tool that can re-create PVCs from these snapshots in the underlying cloud provider. Velero one such tool. (https://github.com/vmware-tanzu/velero)
+1. You have regular backups of the secrets and configmaps in the rook-ceph namespace. Velero provides this functionality too.
+
+Do the following in the new cluster:
+1. Stop the rook operator by scaling the deployment `rook-ceph-operator` down to zero: `kubectl -n rook-ceph scale deployment rook-ceph-operator --replicas 0`
+and deleting the other deployments. An example command to do this is `k -n rook-ceph delete deployment -l operator!=rook`
```

Comment:
Why not delete the operator deployment as well? Scaling it down to zero isn't much different than deleting the deployment. Or else why not scale down all the deployments to 0 instead of deleting them?

Reply:
User convenience. All the other deployments are created and reconciled by operator so once the operator is scaled up they will be restored automatically. The operator itself is installed as part of rook and deleting it will require re-installing rook-ceph (helm, argo, ...).

Reply:
Agreed, when installed with the helm chart you can't just remove the deployment and re-create it easily.